.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Question
+about: Create a question to help us improve our knowledge base and documentation
+labels: question
+---
+
+# Question
+
+Use [Github Discussions](https://github.com/open-telemetry/opentelemetry-dotnet/discussions/new).
```

Comment:
@reyang Discussions are not enabled on this repo and the link points to dotnet. Should we fix the link or enable discussions?

Reply:
I'll send separate PR to fix some of these (discussions, the missing npm config, etc.). We can discuss in that PR whether to use Discussions or Issues.

Reply:
FYI here goes the issue I created yesterday #3.